src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ExceptionDataContract.cs

```diff
@@ -872,5 +872,42 @@ private void Read()
                 throw new InvalidOperationException();
             }
         }
+
+        private string GetEscapedElementString(string stringToEscape)
+        {
+            var sb = new StringBuilder();
```

Comment:
Thanks for the suggestion, Justin. That could've been a better way of writing this. I was trying to catch up the deadline of the next release, so I merged this PR quicker than I normally do. I sent another PR to port this commit to the release\1.0.0 branch. We could consider to change this in the future.